.github/CHANGELOG.md

```diff
@@ -68,8 +68,29 @@
   plt.contourf(xvec, pvec, wigner)
   plt.show()
   ```
+
 * The `fock` backend now supports the `GKP` preparation [(#553)](https://github.com/XanaduAI/strawberryfields/pull/553)
 
+* The `tf` backend now accepts the Tensor DType as argument.
+  [(#562)](https://github.com/XanaduAI/strawberryfields/pull/562)
+
+  Allows high cutoff dimension to give numerically correct calculations:
+
+  ```python
+  prog = sf.Program(2)
+  eng  = sf.Engine("tf", backend_options={"cutoff_dim": 50, "dtype": tf.complex128})
+  with prog.context as q:
+      Sgate(0.8)         | q[0]
+      Sgate(0.8)         | q[1]
+      BSgate(0.5,0.5)  | (q[0], q[1])
+      BSgate(0.5,0.5)  | (q[0], q[1])
+  state    = eng.run(prog).state
+  N0,N0var = state.mean_photon(0)
+  N1,N1var = state.mean_photon(1)
+  print(N0)
+  print(N1)
+  print('analytical:',np.sinh(0.8)**2)
```

Comment:
```suggestion
  state = eng.run(prog).state
  N0, N0var = state.mean_photon(0)
  N1 ,N1var = state.mean_photon(1)
  print(N0)
  print(N1)
  print("analytical:", np.sinh(0.8)**2)
```

Small modification here, as we try to stick to PEP8 formatting for our examples :)